libs/wallet/src/wallet-types.ts

```diff
@@ -229,6 +228,7 @@ interface Buy {
 export interface ProposalSubmission {
   rationale: {
     description: string;
+    title?: string;
     hash?: string;
     url?: string;
```

Comment:
```suggestion
    title?: string;
```

i believe these two fields are also gone here: https://github.com/vegaprotocol/documentation/commit/51002a6bddc5245418d7dba940b7dbb420fc6d01#diff-007cc5bacf747097ad3c6bbe7fc3ee12f453323ef77f5bb9903b0b132da97785L5-L6